docs/api/oauth.md

```diff
@@ -0,0 +1,44 @@
+# OAuth
+
+The API is secured by [OAuth2](https://oauth.net/2/) authorization and we use [Trikoder oauth2-bundle](https://github.com/trikoder/oauth2-bundle) implementation.
+
+We use [Bearer token](https://oauth.net/2/bearer-tokens/) authorization and it means that each API call must have `Authorization: Bearer TOKEN` header.
+
+## Configuration
+
+To configure the OAuth, you have to run [backend-api-oauth-keys-generate](/docs/introduction/console-commands-for-application-management-phing-targets.md#backend-api-oauth-keys-generate) phing target.
+This phing target generates private and public keys for OAuth and also required parameters.
+
+### Client Credentials
+
+OAuth has it's own users and when you want to use the API, you have to create users first.
+
+Run this SQL command to create a user `alan` with secret `xxx`
+
+```sql
+INSERT INTO "oauth2_client" ("identifier", "secret", "grants", "active")
+VALUES ('alan', 'xxx', 'client_credentials password', '1');
+```
+
+> Never use password `xxx` in production, always use secure passwords!
+
+## Try it
+
+Run following code in bash
```

Comment:
I'm missing information about why should I run that or what it does

Reply:
I have updated title